src/api/app/helpers/webui/user_helper.rb

```diff
@@ -1,12 +1,15 @@
 module Webui::UserHelper
-  def display_applied_filters(filters)
+  def display_applied_filters(filters, user)
     tag.span(class: 'd-block ml-4 mb-3') do
-      link_to(User.session!) do
-        tag.i(class: 'fas fa-trash-alt') do
-          tag.b(class: 'pl-2') do
-            "Clear current search #{display_filters(filters.keys).to_sentence}"
-          end
-        end
+      link_to(user) do
+        safe_join(
+          [
+            tag.i(class: 'fas fa-trash-alt'),
```

Comment:
We can remove the `span` and instead add the padding (`pr-2`) to the `i`. The text can be directly outputted.

```suggestion
            tag.i(class: 'fas fa-trash-alt pr-2'),
            "Clear current search #{display_filters(filters.keys).to_sentence}"
```